mllib-dal/src/main/scala/org/apache/spark/mllib/stat/spark320/Statistics.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.mllib.stat.spark320
+
+import com.intel.oap.mllib.Utils
+import com.intel.oap.mllib.stat.{SummarizerDALImpl, SummarizerShim}
+import org.apache.spark.annotation.Since
+import org.apache.spark.api.java.{JavaDoubleRDD, JavaRDD}
+import org.apache.spark.ml.stat._
+import org.apache.spark.mllib.linalg.distributed.RowMatrix
+import org.apache.spark.mllib.linalg.{Matrix, Vector}
+import org.apache.spark.mllib.regression.LabeledPoint
+import org.apache.spark.mllib.stat.MultivariateStatisticalSummary
+import org.apache.spark.mllib.stat.correlation.Correlations
+import org.apache.spark.mllib.stat.test.{ChiSqTest, ChiSqTestResult, KolmogorovSmirnovTest, KolmogorovSmirnovTestResult}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.storage.StorageLevel
+
+import scala.annotation.varargs
+
+
+/**
+  * API for statistical functions in MLlib.
+  */
+@Since("1.1.0")
+class Statistics extends SummarizerShim{
```

Comment:
```suggestion
class Statistics extends SummarizerShim {
```

Reply:
something wrong with CI, pls make sure you didn't mess up other algos.  Only change what you need to change. Pls compare the changed files for 3.1.1, 3.1.2 and 3.2 to see anything inconsistent.